content/en/tests/setup/python.md

```diff
@@ -163,6 +163,213 @@ assert True
 
 {{% /tab %}}
 
+{{% tab "Manual Instrumentation (beta)" %}}
+
+### Using manual testing API
+
+<div class="alert alert-warning"><strong>Note</strong>: The Test Visibility Manual API is in <strong>beta</strong> and subject to change.</div>
+
+As of version `2.13.0`, the [Datadog Python tracer][1] provides the Test Visibility API (`ddtrace.ext.test_visibility`) as a way to submit test visibility results in an ad-hoc manner.
+
+#### API execution
+
+The API uses classes to provide namespaced methods that are intended to be used without the caller maintaining state.
```

Comment:
I'm not sure what "without the caller maintaining state" means here, and I'm not sure if it means much for a user of the library.